src/main/java/com/gtnewhorizons/neid/mixins/Mixins.java

```diff
@@ -0,0 +1,209 @@
+package com.gtnewhorizons.neid.mixins;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+import java.util.function.Supplier;
+
+import com.gtnewhorizons.neid.NEIDConfig;
+
+import cpw.mods.fml.relauncher.FMLLaunchHandler;
+
+public enum Mixins {
```

Comment:
this whole class is overkill, you can just do : 
`getMixins() {
List<String> list = new ArrayList<>();
if (FMLLaunchHandler.side().isClient()) {
list.add ()// client mixins
} else if (FMLLaunchHandler.side().isServer()) {
list.add ()// server mixins
}
list.add ()// bothj sides mixins
return list;
}`